### PR TITLE
Refactoring the Login/Sign to Reusable API Calls and FrontEnd Properties

### PR DESCRIPTION
**Pull request by mepripri**
I have created a folder for all the Signup/Login files. The API functions and properties which will be used entirely is placed in another folder. The magic constants are moved to variables. 
I have also created separate folder for all the images.
Removed some CSS which was not used in the files.

@kishore223 , As we discussed you can add your functions to the functions folder.

Can you please review this PR once, if possible please run it in your local just to double check. :) thanks

### CODE REVIEW

**Comment by mepripri** [comment]
@Sandeep-1-Kumar , As you have suggested the centralising properties idea. Can you also please review this PR.

File: Front-End/src/components/Functions/APIUrl.js
Comment (Sandeep-1-Kumar) [inline Front-End/src/components/Functions/APIUrl.js:1]: Great Work , If in future if we need to modify a URL we now don't need to change at all the files instead we will just change here.

**Comment by kishore223** [comment]
I have executed this code on my local. Good work. And I will add the refactored navbar changes to the function folder.

**Review by kishore223 — APPROVED** [review]

**Review by Sandeep-1-Kumar — APPROVED** [review]
Good work now we cans witch between different hosting platforms just by changing the URL values , W don't need to change every where in code.

**Pull request merged**